### PR TITLE
Removed harcoded version code and made it more dynamic

### LINKED ISSUE #270: Make Version Code Dynamic in settings page.
**Describe the feature you'd like**

Make Version Code Dynamic in the settings page. Currently, it's hardcoded to 1.001

**Would you like to work on the issue?**
No

### PR DESCRIPTION
**Pull request by gconnect**
**What this PR does**
Fixes #270 
- Removed hardcoded version code and made it dynamic.

### CODE REVIEW

**Comment by ujjwalagrawal17** [comment]
Can we use BuildConfig instead of this solution? I think it is cleaner.

Link: https://stackoverflow.com/a/21119027

**Comment by gconnect** [comment]
@ujjwalagrawal17  I have modified it as advice. Pretty neat. Thanks.

File: app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/settings/SettingsActivity.kt [hunk 1]
```diff
@@ -21,7 +25,7 @@ class SettingsActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_settings)
+        setContentView(org.wikiedufoundation.wikiedudashboard.R.layout.activity_settings)
```
Comment (ujjwalagrawal17) [inline app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/settings/SettingsActivity.kt]: org.wikiedufoundation.wikiedudashboard.R can be imported in top to make it more clean.

File: app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/settings/SettingsActivity.kt [hunk 2]
```diff
@@ -30,7 +34,9 @@ class SettingsActivity : AppCompatActivity() {
         textViewPrivacyPolicy.setOnClickListener { openPrivacyPolicy() }
         textViewTermsAndConditions.setOnClickListener { openTermsAndConditions() }
         textViewLogout.setOnClickListener { logOut() }
-        textViewVersionCode.text = "1.001"
+
+        val versionName = BuildConfig.VERSION_NAME
+        textViewVersionCode.text = versionName
```
Comment (ujjwalagrawal17) [inline app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/settings/SettingsActivity.kt]: no need of versionName variable here. Can we do it in 1 line?

File: app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/settings/SettingsActivity.kt [hunk 3]
```diff
@@ -2,15 +2,19 @@ package org.wikiedufoundation.wikiedudashboard.ui.settings
 
 import android.app.AlertDialog
 import android.content.Intent
+import android.content.pm.PackageManager
```
Comment (ujjwalagrawal17) [inline app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/settings/SettingsActivity.kt]: unused import

**Comment by ujjwalagrawal17** [comment]
Looks clean now. Thanks @gconnect

**Pull request merged**